controller/namedspaces_test.go

```diff
@@ -75,7 +76,7 @@ func (rest *TestNamedSpaceREST) TestSuccessQuerySpace() {
 		assert.Fail(t, "Failed to create an identity")
 	}
 
-	name := "Test 24"
+	name := "Test 24" + uuid.NewV4().String()
```

Comment:
oh, interesting! I copied/pasted the test name but this function is much cleaner! On my way to fix that right now \o/